tools/pythonpkg/src/pyconnection.cpp

```diff
@@ -502,6 +500,15 @@ vector<Value> DuckDBPyConnection::TransformPythonParamList(py::handle params) {
 			auto month = PyDateTime_GET_MONTH(ele.ptr());
 			auto day = PyDateTime_GET_DAY(ele.ptr());
 			args.push_back(Value::DATE(year, month, day));
+		} else if (py::isinstance<py::str>(ele)) {
+			args.emplace_back(ele.cast<string>());
+		} else if (py::isinstance<py::memoryview>(ele)) {
+			py::memoryview py_view = ele.cast<py::memoryview>();
+			PyObject* py_view_ptr = py_view.ptr();
+			Py_buffer* py_buf = PyMemoryView_GET_BUFFER(py_view_ptr);
+			args.emplace_back(Value::BLOB(const_data_ptr_t(py_buf->buf), idx_t(py_buf->len)));
+		} else if (py::isinstance<py::bytes>(ele)) {
+			args.emplace_back(Value::BLOB(ele.cast<string>()));
```

Comment:
Creating a `BLOB` from a `string` currently translates `\xnn` sequences to byte values. Correct? `TransformPythonParamList` here needs to just store the string without translation.

Reply:
Yes, in that case you can use the alternative blob interface:
```cpp
//! Create a blob Value from a data pointer and a length: no bytes are interpreted
static Value BLOB(const_data_ptr_t data, idx_t len);
//! Creates a blob by casting a specified string to a blob (i.e. interpreting \x characters)
static Value BLOB(const string &data);
```